### DISCUSSION TITLE
Is it possible to skip the chaptering step and default to "no chapters"?

### DISCUSSION THREAD

**Discussion by Boswell-Scrubbs** [yermak/AudioBookConverter#425] [Q&A]
Huge fan of AudioBookConverter here!

I have a procedural question: When I merge mp3s, I never create chapters (because I don't use them and actually don't want them). Is it possible to add the mp3s and then just click one button to merge them all into an m4b with no chapters?

Right now, if I want to create an m4b with no chapters, I add the mp3, then go to the Chapters tab, highlight all the chapters, combine them, and then click Start. If possible, I would like to just add the mp3, then click Start, but if I do that now, every mp3 becomes a separate chapter in the created m4b. 

Related question: Is combining all the mp3s into one chapter, exactly the same as having no chapters (as with the really old version of this program, "MP3 to iPod Audiobook Converter")? Or is it different -- in other words, is an m4b file created with "one chapter" exactly the same as an m4b file created with "no chapters"?

**Comment by yermak** [comment]
Hi @Boswell-Scrubbs,
Thank you for feedback.

At this point it's not possible to avoid chapters in m4b files. You found the only workaround I could think of - merging everything together, but you are right there still will be ONE chapter.

From quality perspective chapters are just text file embedded into meta information of m4b, which contains title, start, end.
So sound is exactly the same and does not depend on chapters.

Talking about original "MP3 to iPod Audiobook Converter" sound will not be same. Original tool used faac encoder, current program uses ffmpeg with native encoder (earlier versions used Fraunhofer FDK AAC encoder - which is considered the best, but I stopped supporting this due to licensing  limitations of Fraunhofer). I never measured quality vs compressions as for audiobooks it's less important and if you really up for minimal size go for OGG (I use opus coded, which is probably one of the best options for today), not m4b.

It is absolutely possible to add feature of skip chapters creation, could you please explain you use case better.
Unfortunately, I would not add it if your use case is quite unique and won't be massively used.

 Thanks for using AudioBookConverter :)

**Reply by Boswell-Scrubbs** [reply]
Hi Yermak - 

Thank you for your quick and detailed reply.

I guess my use case is just that I don't really use chapters at all. I just listen to an audiobook and stop when I feel like stopping and then start again later at the same spot without regard to chapters. So I liked the quick method in "MP3 to iPod Audiobook Converter" of simply dragging and dropping MP3s into the interface and then clicking Start. I'm not sure if there are many other people who feel the same. Also: I often have mp3 files I convert into an m4b that don't correspond to chapters -- they're just all an arbitrary length so I don't want a chapter mark between every one. Combining them all into one chapter works fine and I'll probably just continue doing that -- and as you say, it probably doesn't matter that I have one chapter for the whole book as compared to no chapters -- same difference really. 

If you did happen to implement a No Chapters option, I was thinking you could do that by putting a "Start - No Chapters" tab just after the current "Start" tab. If a user clicks on that tab, the process is started (asks for m4b file name and location) and the process proceeds to create an m4b file that has no chapters just like with the old "MP3 to iPod Audiobook Converter."

Maybe having this option would reduce support inquiries for you because I think a lot of people might simply skip the "Chapters" and "Start" tabs and go directly to the new "Start - No Chapters" tab (assuming they don't care about chapters like me). Then you would have much fewer queries from those people because that path is very simple for the "No Chapters" type of users.

The Chapters option is very powerful and well implemented in your program for those people who really like using chapters and they would still be able to easily use it just as they always have -- that's a beautiful chapter making interface that you implemented in the Chapters tab!

BTW, thank you for letting me know about the improved encoders you have implemented.

Your program is the best!
